src/component/panels/MoleculesPanel/MoleculePanelHeader.tsx

```diff
@@ -229,6 +231,19 @@ export default function MoleculePanelHeader({
     });
   }
 
+  const topicMolecule = useTopicMolecule();
+
+  function expandMoleculeHydrogens() {
+    const molecule = molecules[currentIndex];
+
+    if (!molecule) return;
+
+    const { id, label } = molecule;
+    const moleculeWithH = topicMolecule[id].moleculeWithH;
```

Comment:
You could also directly use:

https://github.com/cheminfo/openchemlib-utils/blob/main/src/topic/TopicMolecule.ts#L399

And now you can also use 

https://github.com/cheminfo/openchemlib-utils/blob/main/src/topic/TopicMolecule.ts#L407

To get the molfile without H.